crates/network/src/client.rs

```diff
@@ -231,7 +231,11 @@ impl ElementComponent for GameClientView {
                             let game_client =
                                 GameClient::new(conn, Arc::new(create_rpc_registry()), game_state.clone(), client_info.user_id);
 
-                            game_state.lock().world.add_resource(self::game_client(), Some(game_client.clone()));
+                            let world = &mut game_state.lock().world;
```

Comment:
You may find `GameClient::with_physics_world` useful here (especially to disambiguate which world you're working with)

Reply:
Nice helper. I tried to use it but in this case the resulting code is actually more complex. I'd have to:
- destructure `ServerInfo` (`content_base_url` is used in one place and `project_name` in another)
- have additional block with clone for `game_client`